cmd/network-passt-binding/main.go

```diff
@@ -43,6 +43,8 @@ const (
 
 func main() {
 	logFile := flag.String("log-file", passtLogFilePathDefault, "file path for Passt logs")
+	queueCount := flag.Uint("queue-count", 0, "queues count to set for the guest interface")
+	flag.Parse()
```

Comment:
This is odd, I think this was suppose to have happened before, to process the `log-file`.

Reply:
Following offilne discussion we realized a change around Kubevirt hooks API / network binding plugin API is necessary in order to enable Kubevirt pass args to interface binding plugin sidecar in a generic way that would fit any plugin.

Thus the log-file flag is removed at the moment and the path is hard-coded for simplicity.